src/ui/server/helpers/can-access-submit-your-application-routes/index.ts

```diff
@@ -1,17 +1,18 @@
 import flattenApplicationData from '../flatten-application-data';
-import requiredFields from '../required-fields/section-review';
+import requiredFields from '../required-fields/prepare-application';
 import sectionStatus from '../section-status';
 import { TASKS } from '../../content-strings';
 import { Application } from '../../../types';
 
 /**
- * canAccessCheckYourAnswersRoutes
- * Check if an application has submitted previous fields/sections required before accessing the "check your answers" section/routes.
+ * canAccessSubmitYourApplicationRoutes
+ * Check if an application has submitted previous fields/sections required before accessing the "submit your application" group/sections/routes.
+ * The "submit your application" group includes all routes for "check your answers" and "declarations".
```

Comment:
do we not still need one for declarations and one for check your answers?

Reply:
No, this function is only used to check the "prepare application" group so it now consumes both check your answers/section review routes as well as declarations.

Both of these sections for the "submit application" group have the same requirement - both of them can only be unlocked/visited when the "prepare application" group is completed.